shell/platform/android/flutter_main.cc

```diff
@@ -125,6 +125,12 @@ void ConfigureShorebird(std::string android_cache_path,
 
   shorebird_init(&app_parameters, shorebirdYaml.c_str());
 
+  // shorebird_init() copies the paths, so we can free them now.
+  for (auto& path : original_libapp_paths) {
```

Comment:
@eseidel does this look right?

Reply:
It's fine.  I do suspect the more C++ way is to use Vector<char>, but this looks like it's correct.

Reply:
Sounds good, I’ll refactor it 👍

Reply:
One of C++'s many features (but actually useful one) is destructors, especially when applied to stack objects.  In this case they're helpful to you as it makes the memory get automatically cleaned up when the stack object goes out of scope.

Reply:
https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization is a very useful pattern which exploits this in C++.

Reply:
Just pushed the changes lmk what you think and thanks for your help!